### PR TITLE
docs: changelog update additions

### PR DESCRIPTION
**Pull request by SanketD92**
Issue #, if available:
N/A

Description of changes:
Updating changelog for latest changes

Checklist:

- [x] Have you successfully deployed to an AWS account with your changes?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jn1119 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ All notable changes to this project will be documented in this file.
 ## [2.1.4] - 2021-04-06
 
 ### Added
+- chore: Add custom user agent
+- fix: Ensure sdk retry logic is enabled in prod
+- docs: Readme updated
```
Comment (rsmayda) [inline CHANGELOG.md]: These are not in 2.1.4: https://github.com/awslabs/service-workbench-on-aws/commits/v2.1.4

**Review by rsmayda — CHANGES_REQUESTED** [review]
https://github.com/awslabs/service-workbench-on-aws/commits/v2.1.4

File: CHANGELOG.md (hunk 1 above)
Comment (SanketD92) [reply CHANGELOG.md]: Corrected.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ## [2.1.5] - 2021-04-08
 
 ### Added
+- chore: Add custom user agent
```
Comment (rsmayda) [inline CHANGELOG.md]: Typically we shouldn't include `chores` in CHANGELOGs to generate them we can use this tool: https://github.com/conventional-changelog/standard-version

**Review by rsmayda — APPROVED** [review]

**Pull request merged**